js/buttons.js

```diff
@@ -41,10 +41,14 @@ function addButtons() {
       updateCategory(d.toLowerCase(), activeCategory);
     })
     .on('mouseover', function(d){
-      showCategory(d.toLowerCase(), activeCategory, action = 'mouseover');
+      if(waterUseViz.interactionMode === "tap") {
+        showCategory(d.toLowerCase(), activeCategory, action = 'mouseover');
+      }
     })
     .on('mouseout', function(d){
-      showCategory(activeCategory, d.toLowerCase(), action = 'mouseout');
+      if(waterUseViz.interactionMode === "tap") {
+        showCategory(activeCategory, d.toLowerCase(), action = 'mouseout');
+      }
     });
```

Comment:
1. The conditionals you've added make it so there's only hover action if `(waterUseViz.interactionMode === "tap")`...but isn't 'tap' equivalent to 'mobile', where hovering isn't really a thing? Shouldn't you be testing for `==="hover"` instead?
2. The chaining syntax is nice, but this looks to me like a place where breaking the syntax might be cleaner so that you only set any sort of 'mouseover' or 'mouseout' event if hovering is an option. Like this:
```js
  var mouser = buttons.append('rect')
     .classed('mouser-button', true)
     .style('opacity','0')
     .style('stroke-width',1.5)
     .style('pointer-events','all')
     .on('click', function(d){
        updateCategory(d.toLowerCase(), activeCategory);
      });

  if(waterUseViz.interactionMode === "hover") {
    mouser
      .on('mouseover', function(d){
        showCategory(d.toLowerCase(), activeCategory, action = 'mouseover');
      })
      .on('mouseout', function(d){
        showCategory(activeCategory, d.toLowerCase(), action = 'mouseout');
      });
  }
```

Reply:
Oof nice catch on # 1 - really thought about that too and still picked the wrong one. Fixes coming for # 2